testproject/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import url, include
+
+from testproject.views import login, logout
+
+
+urlpatterns = [
+
+    url(r'^i18n/', include('django.conf.urls.i18n')),
+    url(r'^login/', login, name='login'),
+    url(r'^logout/', logout, name='logout'),
```

Comment:
Maybe `logout/` can make use of it but `login/` is a simple bypass that does not require user input, it will just sign in the first user available for testing purposes.